x-pack/plugins/security_solution/public/overview/components/detection_response/user_alerts_table/use_user_alerts_items.ts

```diff
@@ -54,20 +67,35 @@ export const useUserAlertsItems: UseUserAlertsItems = ({ skip, queryId, signalIn
     response,
     refetch: refetchQuery,
   } = useQueryAlerts<{}, AlertCountersBySeverityAggregation>({
-    query: buildVulnerableUserAggregationQuery({ from, to }),
+    query: buildVulnerableUserAggregationQuery({
+      from,
+      to,
+      currentPage: paginationData.currentPage,
+    }),
     indexName: signalIndexName,
     skip,
   });
 
   useEffect(() => {
-    setQuery(buildVulnerableUserAggregationQuery({ from, to }));
-  }, [setQuery, from, to]);
+    setQuery(
+      buildVulnerableUserAggregationQuery({ from, to, currentPage: paginationData.currentPage })
+    );
+  }, [setQuery, from, to, paginationData.currentPage]);
 
   useEffect(() => {
     if (data == null || !data.aggregations) {
       setItems([]);
     } else {
       setItems(parseUsersData(data.aggregations));
+
+      setPaginationData((p) => ({
+        ...p,
+        pageCount: Math.ceil(
```

Comment:
Not necessary, but you might be able to surface this to a util getPageCount which can be used here and in the hosts one

Reply:
fixed :D